tools/rosbash/scripts/rosrun

```diff
@@ -66,15 +68,18 @@ if [[ ! $2 == */* ]]; then
     _perm="/111"
   fi
   debug "Searching for $2 with permissions $_perm"
-  exepathlist=(`find -L $catkin_package_libexec_dir $pkgdir -name $2 -type f  -perm $_perm ! -regex ".*$pkgdir\/build\/.*" | uniq`)
+  exepathlist="`find -L "${catkin_package_libexec_dirs[@]}" "$pkgdir" -name "$2" -type f  -perm "$_perm" ! -regex ".*$pkgdir\/build\/.*" | uniq`"
```

Comment:
Running the `find` command manually fails. Within the `rosrun` script is seems to deal with the fact that two paths are quotes as one...